doc/api.rst

```diff
@@ -195,6 +195,13 @@ of strings bound to template names::
 This loader is very useful for unit testing. It can also be used for small
 projects where storing all templates in a single PHP file might make sense.
 
+``Twig_Loader_Array`` can also be used in conjuction with ``Twig_Environment``
+to create a template from a string::
+
+    $env = new \Twig_Environment(new \Twig_Loader_Array());
+    $template = $env->createTemplate('Hello, {{ name }}');
+    echo $env->render($template, array('name' => 'Bob')); // Hello, Bob
```

Comment:
according to the phpdoc of the Twig environment this is not allowed. `render` expects a string, but `createTemplate` returns a Twig_Template. so if this is the recommended way of doing things, then the phpdocs must be updated.